test/unit/BPool/BPool_JoinPool.t.sol

```diff
@@ -55,28 +65,26 @@ contract BPoolJoinPool is BPoolBase {
     bPool.call__pullPoolShare(deployer, INIT_POOL_SUPPLY);
     bPool.call__burnPoolShare(INIT_POOL_SUPPLY);
     // it should revert
+    // division by zero
```

Comment:
just to keep it same accross files
```suggestion
    // it should revert
    //     division by zero
```

Reply:
I tried to set it as a sub-case of `it should revert`, which would cause this indent, but that caused the expected test function name to change

perhaps we have inconsistencies in existing test names because of this?

Reply:
i think the last bulloak version should fix this, i haven't checked it tho
https://github.com/alexfertel/bulloak/issues/69

Reply:
Hey! Circling back here. Did this get resolved?